src/tools/pkcs11-tool.c

```diff
@@ -5363,6 +5482,16 @@ static int test_decrypt(CK_SESSION_HANDLE sess)
 		printf("No OpenSSL support, unable to validate decryption\n");
 #else
 		for (n = 0; n < num_mechs; n++) {
+			switch (mechs[n]) {
+			case CKM_RSA_PKCS:
+			case CKM_RSA_PKCS_OAEP:
+			case CKM_RSA_X_509:
+				break;
+			default:
+				printf(" -- mechanism can't be used to decrypt, skipping\n");
```

Comment:
This output is not helpful since it does not show what mechanism can not be used to decrypt. You should print  at least the mechanism name before this message, ideally using `p11_mechanism_to_name().`

Reply:
I improved the output as suggested in my new changes.

Reply:
I think that's not pushed to the PR, is it?